### ISSUE TITLE
Linked image missing alt text

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/wp-proudcity#2326]
## Source

LF

## Describe the bug

The contacts widget has an accessibility 'Linked image missing alt text issue. 

## Screenshots

![Screenshot 2023-08-02 at 8 25 23 AM](https://github.com/proudcity/wp-proudcity/assets/1569463/248184d0-067f-4ec4-a8f4-915e3768a241)

## QA

Notes:

* 

Links:

*

**Comment by curtismchale** [comment]
@lukefretwell This works fine if the user puts in `alt` text. With the new ALT text plugin these should get resolved.
